app/views/hyrax/dashboard/collections/edit.html.erb

```diff
@@ -4,33 +4,9 @@
   <h1><span class="fa fa-edit"></span><%= t('.header', type_title: @collection.collection_type.title, title: @form.title.first) %></h1>
 <% end %>
 
-<% unless has_collection_search_parameters? %>
-  <div class="row">
-    <div class="col-md-12">
-      <%= render 'flash_msg' %>
-      <%= render 'form' %>
-    </div>
+<div class="row">
+  <div class="col-md-12">
+    <%= render 'flash_msg' %>
+    <%= render 'form' %>
   </div>
-
-  <div class="row">
-    <div class="col-md-12">
-      <div class="panel panel-default">
-        <div class="panel-body">
-          <%= render 'edit_actions' %>
-
-          <div class="row m-t-3">
-            <h2 class="col-xs-12 col-sm-6"><%= t('hyrax.collection.edit.manage_items') %></h2>
-            <div class="col-xs-12 col-sm-6">
-              <%= render 'hyrax/collections/search_form', presenter: @collection %>
-            </div>
-          </div>
-          <%= render 'hyrax/my/did_you_mean' %>
-          <%= render 'hyrax/my/facet_selected' %>
-          <%= render 'sort_and_per_page', collection: @collection if @response.response['numFound'] > 0 %>
```

Comment:
Can `app/views/hyrax/collections/_sort_and_per_page.html.erb` be removed now?

Reply:
I think it's still used on the public collections show page, but I'll be double checking as I do the cleanup.

Reply:
You are right `app/views/hyrax/collections/_sort_and_per_page.html.erb` is used by the public collections page. Sorry, I meant to ask if `app/views/hyrax/dashboard/collections/_sort_and_per_page.html.erb` could be removed.